content/docs/user-guide/assets/asset-processor/debugging.md

```diff
@@ -122,3 +122,107 @@ If you're a game artist and you're having issues running Asset Processor, the is
 {{< note >}}
 If you're an engineer making new BuilderSDK-based builders, we recommend that you don't delete your cache.
 {{< /note >}}
+
+
+## Debugging the Scene Pipeline
+
+### Situation 
+
+The scene pipeline imports source scene assets into a scene graph that contains the scene nodes like meshes and materials. The scene manifest adds processing rules that scene builders use to output scene product assets like models, collision meshes, and animations. It can be frustrating to determine what the scene pipeline found in the source scene file, how it decomposes the scene into a scene graph, and what scene manifest rules it used to generate product assets.
+
+### Effect
+
+The frustration can lead to the scene author to tweak minor data in the original source scene (i.e. the Blender file) and re-exporting to attempt to resolve strange errors from the O3DE scene pipeline. The scene pipeline does many processing steps so it can be confusing to determine which the scene node data (e.g. Transform Data, Mesh Data) were discovered and the rules used to import the scene nodes. The scene pipeline events can be overridden by either Python scripts or C++ code to modify the scene manifest rules. This can lead to confusion of what rules were used to generate the product assets.

```

Comment:
maybe add a "of" between which and the? "The scene pipeline does many processing steps so it can be confusing to determine which **of** the scene node data"